backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -73,7 +74,14 @@ object VeloxBackendSettings extends BackendSettingsApi {
       format: ReadFileFormat,
       fields: Array[StructField],
       partTable: Boolean,
+      rootPaths: Seq[String],
       paths: Seq[String]): ValidationResult = {
+    if (
+      !rootPaths.isEmpty && !VeloxFileSystemValidationJniWrapper.supportedPaths(rootPaths.toArray)
```

Comment:
Can we just extract the scheme to validate? For example, if `rootPaths consists of multiple HDFS files, we only need to let scheme "hdfs://" be validated on native side, not one file by one file validation.

In addition, as local file system is always supported by Gluten + Velox. Maybe, we can have a fast path to directly let validation pass on scala side for such case.

Reply:
> Can we just extract the scheme to validate?

Velox API is designed to accept file path, now looks all file systems validate the schema only. But I think it's better to align this with the velox api to pass the file path. Now only one file path is validated.

> In addition, as local file system is always supported by Gluten + Velox. Maybe, we can have a fast path to directly let validation pass on scala side for such case.

Sure.

Reply:
@zhli1142015, I note Spark accepts data from two or more kinds of file systems, e.g.,

`df = spark.read.format("parquet").load("file:///path/to/local/data", "s3a://bucket/path/to/s3/data")`.
 
Not sure whether we need to also consider other files other than the first one, which definitely brings some cost. Does your production env. have such usage? If it is rarely used, we can just put some comments to clarify. cc @wForget

Reply:
I see, we don't see such usage, but let's only validate one path fpr each schema.

Reply:
We don't have this usage, but I'd favor limiting it, perhaps by simply using `rootPaths.take(10)`